internal/versionsapi/cli/list.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/internal/logger"
+	"github.com/edgelesssys/constellation/v2/internal/versionsapi"
+	verclient "github.com/edgelesssys/constellation/v2/internal/versionsapi/client"
+	"github.com/rogpeppe/go-internal/semver"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap/zapcore"
+)
+
+func newListCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List versions",
+		Long:  "List all versions of a ref/stream. The returned version are in short format, if --json flag is not set.",
+		RunE:  runList,
+		Args:  cobra.ExactArgs(0),
+	}
+
+	cmd.Flags().String("ref", "-", "Ref to query")
```

Comment:
```
./versionsapi latest --ref /../
v2.3.0
```
I feel I could do some interesting stuff here 😆 not sure if this is really critical though for internal tooling

Reply:
:clap: :clap: :clap: